pkg/dns/vips_allocator_test.go

```diff
@@ -159,7 +170,12 @@ var _ = Describe("VIP Allocator", func() {
 
 	It("should try to update all meshes and return combined error", func() {
 		errConfigManager := &errConfigManager{ConfigManager: cm}
-		errAllocator, err := dns.NewVIPsAllocator(rm, errConfigManager, true, "240.0.0.0/24")
+		config := dns_server.DNSServerConfig{
```

Comment:
nit: can this config change? can we create it once and reuse it in these 3 tests?

Reply:
done, also renamed the config struct